Source/WebCore/platform/mediastream/mac/BaseAudioSharedUnit.cpp

```diff
@@ -353,6 +356,8 @@ void BaseAudioSharedUnit::voiceActivityDetected()
     if (m_voiceActivityThrottleTimer.isActive() || !m_voiceActivityCallback)
         return;
 
+    RELEASE_LOG_ERROR(WebRTC, "BaseAudioSharedUnit::voiceActivityDetected");
```

Comment:
Should this really be logged as an error?

Reply:
Right, fixed.